NOTES.WIN

```diff
@@ -53,6 +53,9 @@
    64-bit version is exercised through continuous integration of
    VC-WIN64A-masm target.
 
+ If you want to target non-default, older Windows version, just pass
+ additional -D_WIN32_WINNT=0xNml flag to ./Configure. For example pass
+ -D_WIN32_WINNT=0x501 to target XP.
```

Comment:
What is the minimum version that we support? Perhaps we should document that here.

Reply:
Trick question. Thing to recognize is that mingw is supported and it's rather XP-y. Well, it's more 2003-y than XP-y, but either way, as long as mingw is supported there is no reason to believe that VC-WIN32 -D_WIN32_WINNT=0x501 wouldn't work. Modulo vendor-provided headers of course, but there is no reason to believe that there is no user who wouldn't have working headers. At least VS2017 headers work and binaries compiled on Windows 10 do run on XP. As well as on Windows 10 itself...

As for Appveyor. As already said, mingw is 2003-y, so that legacy-specific problems should be caught on Travis. One can still make a case for exercising specifically VC build. Well, trouble is that Appveyor is rather slow, and you probably don't want to increase the matrix, rather to modify one of current ones. But if one aims to cover most with current set, I for one would say there is more interesting case to cover, namely no-shared --debug.